pkg/component/gardener/apiserver/apiserver.go

```diff
@@ -59,6 +59,8 @@ type Values struct {
 	Autoscaling AutoscalingConfig
 	// ClusterIdentity is the identity of the garden cluster.
 	ClusterIdentity string
+	// FastDeployment reduces min ready and termination grace periods for faster non-productive deployments.
+	FastDeployment bool
```

Comment:
I think we should rather let `provider-local` mutate the fields to speed up the process in local dev and e2e tests

Reply:
FYI: We opened https://github.com/gardener/gardener/pull/12156